proto/scheduler.proto

```diff
@@ -107,6 +107,18 @@ message TaskSize {
   // The task's estimated disk space requirement (beyond task inputs).
   int64 estimated_free_disk_bytes = 3;
 
+  // The task's estimated disk bytes written per second.
+  int64 estimated_disk_wbps = 5;
+
+  // The task's estimated disk bytes read per second.
+  int64 estimated_disk_rbps = 6;
+
+  // The task's estimated disk read IO operations per second.
+  int64 estimated_disk_riops = 7;
+
+  // The task's estimated disk write IO operations per second.
+  int64 estimated_disk_wiops = 8;
```

Comment:
nit: all fields in here start with estimated_ which seems sort of redundant

Reply:
yeah, it's a pre-existing issue, but I got rid of `estimated_` for now and we can rename the other fields later (should be wire-compatible)